src/themes/dspace/app/navbar/navbar.component.scss

```diff
@@ -1,15 +1,19 @@
+@font-face {
+  font-family: 'TUL Mono';
```

Comment:
tul.cz has in the navbar `0.85rem 'TUL Mono', sans-serif;` is it the same?

Reply:
I'll add it